RubeesEat.IntegrationTests/WebTesting/PageObject.cs

```diff
@@ -26,6 +28,47 @@ public IReadOnlyCollection<IWebElement> FindElements(By by)
     {
         return Driver.FindElements(by);
     }
+    
+    public IWebElement FindElement(string data_test_id)
+    {
+        ReadOnlyCollection<IWebElement> elements = Driver.FindElements(By.CssSelector($"[data-test-id='{data_test_id}']"));
+        if (elements.Count == 0)
+        {
+            throw new NoSuchElementException($"Element with id: {data_test_id} was not found.");
+        }
+
+        if (elements.Count > 1)
+        {
+            throw new TooManyElementsException();
+        }
+        return elements.First();

```

Comment:
If you are going to change the code above to single line ifs (as joel suggested) you should do it here as well